.github/workflows/lint.yml

```
@@ -24,7 +24,5 @@ jobs:
       - name: Lint with flake8
         run: |
           pip install flake8
-          # stop the build if there are Python syntax errors or undefined names
-          flake8 pywebio --count --select=E9,F63,F7,F82 --show-source --statistics
-          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-          flake8 pywebio --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
\ No newline at end of file
+          flake8 --ignore=E126,E127,E128,E131,E226,E231,E302,E303,E305,E401,E402,E701,E731,F401,F403,F405,W291,W292,W293,W391 \
+                 --max-complexity=32 --max-line-length=525 --show-source --statistics .
```

I think I need to adjust my code first, to conform to the flake8 rules as much as possible.
So it's better to keep it as it is.
```suggestion
          # stop the build if there are Python syntax errors or undefined names
          flake8 pywebio --count --select=E9,F63,F7,F82 --show-source --statistics
          # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
          flake8 pywebio --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

---

This test passes.  Basically, this puts up a fence so no one can create new problems that are not on this list.  Then we can do PRs that fix a handful of codes at a time and remove them from the list.  It is better to catch issues than to exit-zero them all the time.

Running `psf/black` will autofix many of these issues.